libraries/Microsoft.Bot.Connector/Authentication/JwtTokenValidation.cs

```diff
@@ -25,8 +26,8 @@ public static class JwtTokenValidation
         /// validation may require outbound calls for Endorsement validation and other checks. Those calls are made to
         /// TLS services, which are (latency wise) expensive resources. The httpClient passed in here, if shared by the layers
         /// above from call to call, enables connection reuse which is a significant performance and resource improvement.</param>
-        /// <returns>Nothing</returns>
-        public static async Task AssertValidActivity(Activity activity, string authHeader, ICredentialProvider credentials, HttpClient httpClient = null)
+        /// <returns>Claims identity. An anonymous identity is returned if Authentication is disabled.</returns>
+        public static async Task<ClaimsIdentity> EnsureValidActivity(IActivity activity, string authHeader, ICredentialProvider credentials, HttpClient httpClient = null)
```

Comment:
The name of this method is "EnsureValid". having it return something doesn't make sense. The intent of this method is to just be slient if things are good, and throw if things are bad. 

*IF* we want to return the ClaimsIdentity, we'll need to change the method name.